src/app.js

```diff
@@ -78,16 +75,28 @@ const MONGODB_URI =
         : process.env.DEVELOPMENT_DB_CONNECTION
 
 // Database connection
-mongoose
-    .connect(MONGODB_URI, {
-        useNewUrlParser: true,
-        useUnifiedTopology: true,
-    })
-    .then(() => console.log("DB Connected "))
-    .catch((err) => console.log("error"))
-
-if(process.env.NODE_ENV !== "test") {  
-app.listen(process.env.PORT, () => console.log(`App listening at port ${process.env.PORT}`));
-};
+if (process.env.MONGODB_URI) {
+    mongoose
```

Comment:
I added `process.env.MONGODB_URI`

I also created this larger `if/else` statement, to handle the multiple environments (production, dev, test)